packages/aws-fargate/src/metrics/processorRegistry.js

```diff
@@ -21,23 +17,30 @@ const SecondaryDockerProcessor = require('./docker/SecondaryDockerProcessor');
 const allProcessors = [];
 
 exports.init = function init(config, metadataUri, onReady) {
-  coreAndShared.init(config);
-
   const oneMinute = 60 * 1000;
-  const metadataRootDataSource = new HttpDataSource(metadataUri, oneMinute);
-  const metadataTaskDataSource = new HttpDataSource(`${metadataUri}/task`, oneMinute);
-  const metadataRootStatsDataSource = new HttpDataSource(`${metadataUri}/stats`);
-  const metadataTaskStatsDataSource = new HttpDataSource(`${metadataUri}/task/stats`);
 
+  // Common processors from metrics-util
+  const metadataRootDataSource = new metricsUtil.HttpDataSource(metadataUri, oneMinute);
+  const metadataTaskDataSource = new metricsUtil.HttpDataSource(`${metadataUri}/task`, oneMinute);
+  const metadataRootStatsDataSource = new metricsUtil.HttpDataSource(`${metadataUri}/stats`);
+  const metadataTaskStatsDataSource = new metricsUtil.HttpDataSource(`${metadataUri}/task/stats`);
+  const coreAndSharedMetricsDataSource = new metricsUtil.nodejs.CoreDataSource(config);
```

Comment:
Instantiate "CoreDataSource" here and pass it into the target processor "NodeJsProcessor".
Consistency change. See e.g. "InstrumentedDockerProcessor".